client/components/phone-input/index.jsx

```diff
@@ -24,6 +24,7 @@ const PhoneInput = React.createClass( {
 	},
 
 	getCountry( countryCode = this.props.countryCode ) {
+		countryCode = countryCode.toLowerCase();
```

Comment:
I removed all `toUpperCase()` calls within `phone-input.jsx`, I'm converting them to uppercase in `form-phone-media-input.jsx` only once.